addon/services/user-activity.js

```diff
@@ -46,14 +45,13 @@ export default Service.extend(Evented, {
   init() {
     this._super(...arguments);
 
-    if (testing) { // Do not throttle in testing mode
-      this.set('EVENT_THROTTLE', 0);
-    }
-    this.setProperties({
-      _boundEventHandler: this.handleEvent.bind(this),
-      _eventsListened: A(),
-      _throttledEventHandlers: {}
-    });
+    // Do not throttle in testing mode
+    this._EVENT_THROTTLE = testing ? 0 : 100;
```

Comment:
This is not intended to be a private variable, and it should handle a consumer-provided default. IE

```
UserActivityService.extend({
  EVENT_THROTTLE: 300
});
```